torchdynamo/allowed_functions.py

```diff
@@ -113,7 +113,10 @@ def _find_torch_objects(module):
                     if obj.__name__.startswith("torch."):
                         torch_object_ids[id(obj)] = f"{module.__name__}.{name}"
                         _find_torch_objects(obj)
-                else:
+                elif inspect.getmodule(obj) and (
+                    inspect.getmodule(obj).__name__.startswith("torch")
+                    or inspect.getmodule(obj).__name__.startswith("math")
+                ):
```

Comment:
`.startswith` supports a tuple iirc, so we can do:
```
allowed_module_name_prefix_list = ["torch", "math"]
```
...
```
inspect.getmodule(obj).__name__.startswith(tuple(allowed_module_name_prefix_list))
```

Reply:
Nice!